client/signup/steps/theme-selection/index.jsx

```diff
@@ -50,60 +48,57 @@ const ThemeSelectionStep = React.createClass( {
 
 		SignupActions.submitSignupStep( {
 			stepName: this.props.stepName,
-			processingMessage: this.translate( 'Adding your theme' ),
+			processingMessage: this.props.translate( 'Adding your theme' ),
 			repoSlug
 		}, null, {
 			theme: repoSlug
 		} );
 
 		this.props.goToNextStep();
-	},
+	};
 
-	handleScreenshotClick( theme ) {
-		this.pickTheme( theme );
-	},
-
-	handleThemeUpload() {
+	handleThemeUpload = () => {
 		this.setState( {
 			showPressable: true
 		} );
 
 		this.scrollUp();
-	},
+	};
 
 	renderThemesList() {
 		return (
 			<SignupThemesList
 				surveyQuestion={ this.props.chosenSurveyVertical }
 				designType={ this.props.designType || this.props.signupDependencies.designType }
-				handleScreenshotClick={ this.handleScreenshotClick }
+				handleScreenshotClick={ this.pickTheme }
 				handleThemeUpload={ this.handleThemeUpload }
 			/>
 		);
-	},
+	}
 
 	renderJetpackButton() {
 		return (
-			<Button compact href="/jetpack/connect">{ this.translate( 'Or Install Jetpack on a Self-Hosted Site' ) }</Button>
+			<Button compact
```

Comment:
Could we reformat this so the closing tag is on a new line? Seems more readable that way to me:

```javascript
<Button compact href="/jetpack/connect">
    { this.props.translate( 'Or Install Jetpack on a Self-Hosted Site' ) }
</Button>
```